acct-user/monitorix/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>dwosky@pm.me</email>
+		<name>Pedro Arizmendi</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+</pkgmetadata>
```

Comment:
Github shows these errors in your files, I'm not really sure what they mean, but probably something with your editor. Could you check?

Reply:
I think it's the missing newline at the end.

Reply:
I mean, yes, that much was obvious, but I don't remember how to fix these. I had the same error when I used emacs. So I'd say it's something in your editor's configuration.

Reply:
Not sure why it was like that, I've checked both user and group file and should be corrected.